### PR TITLE
Chore: release notes 2021.12

### LINKED ISSUE #6323: release notes 2021.12
release notes 2021.12

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #6323

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,88 @@ id: Release notes
...
+- [@patternfly/react-log-viewer@4.6.12](https://www.npmjs.com/package/@patternfly/react-log-viewer/v/4.6.12)
+- [@patternfly/react-styles@4.11.16](https://www.npmjs.com/package/@patternfly/react-styles/v/4.11.16)
+- [@patternfly/react-table@4.30.1](https://www.npmjs.com/package/@patternfly/react-table/v/4.30.1)
+- [@patternfly/react-tokens@4.12.18](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.12.18)
+- [@patternfly/react-topology@4.9.78](https://www.npmjs.com/package/@patternfly/react-topology/v/4.9.78)
+- [@patternfly/react-virtualized-extension@4.9.47](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.9.47)
+
+### Components
+- **Alert:**
+  - Added plain variant, add expandable inline ([#6236](https://github.com/patternfly/patternfly-react/pull/6236))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:22]: Change "add" to past tense "added"
```suggestion
  - Added plain variant, added expandable inline ([#6236](https://github.com/patternfly/patternfly-react/pull/6236))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,88 @@ id: Release notes
...
+- **Avatar:**
+  - Convert examples to typescript ([#6228](https://github.com/patternfly/patternfly-react/pull/6228))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:24]: Change "Convert" to past tense "Converted"

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -3,6 +3,88 @@ id: Release notes
...
+- **Back to top:**
+  - Introduced Back to Top component and demos ([#6282](https://github.com/patternfly/patternfly-react/pull/6282))
+- **Calendar month:**
+  - Make useEffect respect validators ([#6315](https://github.com/patternfly/patternfly-react/pull/6315))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:28]: Change "Make" to past tense "Made"
```suggestion
  - Made useEffect respect validators ([#6315](https://github.com/patternfly/patternfly-react/pull/6315))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -3,6 +3,88 @@ id: Release notes
...
+- [@patternfly/react-topology@4.9.78](https://www.npmjs.com/package/@patternfly/react-topology/v/4.9.78)
+- [@patternfly/react-virtualized-extension@4.9.47](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.9.47)
+
+### Components
+- **Alert:**
+  - Added plain variant, add expandable inline ([#6236](https://github.com/patternfly/patternfly-react/pull/6236))
+- **Avatar:**
+  - Convert examples to typescript ([#6228](https://github.com/patternfly/patternfly-react/pull/6228))
+- **Back to top:**
+  - Introduced Back to Top component and demos ([#6282](https://github.com/patternfly/patternfly-react/pull/6282))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:26]: Fix capitalization of component name - "back to top"
```suggestion
  - Introduced back to top component and demos ([#6282](https://github.com/patternfly/patternfly-react/pull/6282))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 5]
```diff
@@ -3,6 +3,88 @@ id: Release notes
...
+- **Select:**
+  - Fixed select field dropdown showing the create option even when there's no input ([#6248](https://github.com/patternfly/patternfly-react/pull/6248))
+  - Added aria-label to toggle ([#6271](https://github.com/patternfly/patternfly-react/pull/6271))
+- **Slider:**
+  - Fixed customSteps spacing to respect min/max ([#6231](https://github.com/patternfly/patternfly-react/pull/6231))
+  - Updated slider to respect min/max constraints ([#6232](https://github.com/patternfly/patternfly-react/pull/6232))
+- **Spinner:**
+  - Fixed spinner tests to check all component sizes ([#6269](https://github.com/patternfly/patternfly-react/pull/6269))
+- **Tabs:**
+  - Added Support for disabled state tabs ([#6261](https://github.com/patternfly/patternfly-react/pull/6261))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:64]: Change "Support" to lowercase
```suggestion
  - Added support for disabled state tabs ([#6261](https://github.com/patternfly/patternfly-react/pull/6261))
```

**Review by evwilkin — CHANGES_REQUESTED** [review]
A handful of nits, mostly tense and capitalization tweaks

**Review by evwilkin — DISMISSED** [review]
LGTM - thanks! 🔥

**Review by redallen — APPROVED** [review]

File: packages/react-docs/RELEASE-NOTES.md [hunk 6]
```diff
@@ -3,6 +3,88 @@ id: Release notes
...
+- [@patternfly/react-code-editor@4.3.59](https://www.npmjs.com/package/@patternfly/react-code-editor/v/4.3.59)
+- [@patternfly/react-core@4.157.1](https://www.npmjs.com/package/@patternfly/react-core/v/4.157.1)
+- [@patternfly/react-icons@4.11.17](https://www.npmjs.com/package/@patternfly/react-icons/v/4.11.17)
+- [@patternfly/react-inline-edit-extension@4.7.82](https://www.npmjs.com/package/@patternfly/react-inline-edit-extension/v/4.7.82)
+- [@patternfly/react-log-viewer@4.6.12](https://www.npmjs.com/package/@patternfly/react-log-viewer/v/4.6.12)
+- [@patternfly/react-styles@4.11.16](https://www.npmjs.com/package/@patternfly/react-styles/v/4.11.16)
+- [@patternfly/react-table@4.30.1](https://www.npmjs.com/package/@patternfly/react-table/v/4.30.1)
+- [@patternfly/react-tokens@4.12.18](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.12.18)
+- [@patternfly/react-topology@4.9.78](https://www.npmjs.com/package/@patternfly/react-topology/v/4.9.78)
+- [@patternfly/react-virtualized-extension@4.9.47](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.9.47)
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:18]: You'll have to change these in a second.

**Review by redallen — CHANGES_REQUESTED** [review]
Product testing came back negative, had to merge #6325

**Review by evwilkin — COMMENTED** [review]
LGTM

**Review by redallen — APPROVED** [review]
Thanks for updating the versions!

**Pull request merged**